app/_hub/kong-inc/mtls-auth/_index.md

```diff
@@ -155,42 +155,41 @@ params:
+  case the anonymous consumer is used and the request is allowed to proceed.
 
 
-### Client Certificate request
+### Client certificate request
 Client certificates are requested in the `ssl_certificate_by_lua` phase where Kong does not
-have access to `route` and `workspace` information. Due to this information gap, Kong will ask for
-the client certificate on every handshake if the `mtls-auth` plugin is configured on any Route or Service.
+have access to `route` and `workspace` information. Due to this information gap, Kong asks for
+the client certificate by default on every handshake if the `mtls-auth` plugin is configured on any route or service.
 In most cases, the failure of the client to present a client certificate is not going to affect subsequent
-proxying if that Route or Service does not have the `mtls-auth` plugin applied. The exception is where
-the client is a desktop browser, which will prompt the end user to choose the client cert to send and
+proxying if that route or service does not have the `mtls-auth` plugin applied. The exception is where
+the client is a desktop browser, which prompts the end user to choose the client cert to send and
 lead to user experience issues rather than proxy behavior problems. To improve this situation,
-Kong builds an in-memory map of SNIs from the configured Kong Routes that should present a client
+Kong builds an in-memory map of SNIs from the configured Kong routes that should present a client
 certificate. To limit client certificate requests during handshake while ensuring the client
-certificate is requested when needed, the in-memory map is dependent on all the Routes in
+certificate is requested when needed, the in-memory map is dependent on the routes in
 Kong having the SNIs attribute set. When any routes do not have SNIs set, Kong must request
 the client certificate during every TLS handshake:
 
 - On every request irrespective of Workspace when the plugin is enabled in global Workspace scope.
-- On every request irrespective of Workspace when the plugin is applied at the Service level
-  and one or more of the Routes *do not* have SNIs set.
-- On every request irrespective of Workspace when the plugin is applied at the Route level
-  and one or more Routes *do not* have SNIs set.
-- On specific requests only when the plugin is applied at the Route level and all Routes have SNIs set.
+- On every request irrespective of Workspace when the plugin is applied at the service level
+  and one or more of the routes *do not* have SNIs set.
+- On every request irrespective of Workspace when the plugin is applied at the route level
+  and one or more routes *do not* have SNIs set.
+- On specific requests only when the plugin is applied at the route level and all routes have SNIs set.
 
-The result is all Routes must have SNIs if you want to restrict the handshake request
-for client certificates to specific requests.
+SNIs must be set for all routes that mtls-auth was meant to be effective.
```

Comment:
@ms2008 Just to make sure I'm understanding this correctly, another way of saying this is that SNIs must be set for all routes that mutual TLS authentication uses? I think the "was meant to be effective" part is tripping me up because I'm not sure what is meant to be effective.

Reply:
Looks great, and feels more fluid. 👍